lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,7 +15,7 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
-  $date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
Yeah, this is used to toggle [how client-side validation validates](https://github.com/DoSomething/phoenix/blob/1798036e74112e05af06cf67f71568e8e2707fde/lib/themes/dosomething/paraneue_dosomething/js/validators/auth.js#L87). Should probably keep as is!